### PR TITLE
added upterm under command line tools

### PR DESCRIPTION
**Pull request by tarungulati1988**
- Upterm is a terminal based on electron and black screen and when clubbed with oh my zshell it becomes extremely powerful.
- Added it under Command Line tools

### CODE REVIEW

**Comment by xcv58** [comment]
Thanks @tarungulati1988 !

We already have Black Screen in https://github.com/jaywcjlove/awesome-mac#command-line-tools. 
Could you please use Upterm to replace Black Screen?

**Comment by xcv58** [comment]
Just my personal opinion: a terminal can NOT `cd` doesn't deserve awesome: https://github.com/railsware/upterm/issues/462

**Comment by tarungulati1988** [comment]
I am not sure about the cd issue as i have been using this terminal heavily for about 8-10 months and i haven't seen any such issue. This change was pushed using upterm.

File: README.md [hunk 1]
```diff
@@ -181,6 +181,7 @@ for the 21st century made by GitHub. [Atom Plugins](editor-plugin.md#atom-plugin
 * [Serial](https://www.decisivetactics.com/products/serial/) - Full-featured serial terminal for the Mac.
 * [ttygif](https://github.com/icholy/ttygif) - 将终端录制转换为GIF动画。 [![Open-Source Software][OSS Icon]](https://github.com/icholy/ttygif) ![Freeware][Freeware Icon]
 
+
```
Comment (xcv58) [inline README.md]: Could you please remove the extra blank line.

**Pull request merged**